packaging/leapp.spec

```
@@ -21,15 +21,16 @@ Summary:    OS & Application modernization framework
 License:    ASL 2.0
 URL:        https://leapp-to.github.io
 Source0:    https://github.com/leapp-to/%{name}/archive/%{gittag}/%{name}-%{version}.tar.gz
-
 BuildArch:  noarch
+
+%if !0%{fedora}
```

Eror: /tmp/temp/packaging/leapp.spec:26: bad %if condition

```suggestion
%if !0%{?fedora}
```